zkdoc/release-note

```diff
@@ -55,6 +73,7 @@ ZK 9.6.3
   ZK-5140: ZK loads vulnerable commons-io transitively
   ZK-5223: the if element doesn't keep the position of its child component
   ZK-5116: NullPointerException happened during evaluating an EL expression
+  ZK-3543: Can't wire <apply> with its ID
```

Comment:
Weird release note

Reply:
For the release-note and config.properties, I followed the order in 9.6.3, so the diff between every commit with head of master will show the progress.